eZ/Publish/API/Repository/URLAliasService.php

```diff
@@ -134,4 +134,18 @@ public function reverseLookup(Location $location, $languageCode = null);
      * @return \eZ\Publish\API\Repository\Values\Content\URLAlias
      */
     public function load($id);
+
+    /**
+     * Refresh all system URL aliases for the given Location (and historize outdated if needed).
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     */
+    public function refreshSystemUrlAliasesForLocation(Location $location);
```

Comment:
Perhaps we should also add `void` as a return type here?

Reply:
Fixed via 9847feb